### PR TITLE
chore: remove Select hover undefined line width

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

看上去是改漏了。不过没看出原来的代码有啥用。

图片

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   --        |
| 🇨🇳 Chinese |  --         |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by li-jia-nan — APPROVED** [review]

File: components/select/style/index.tsx [hunk 1]
```diff
@@ -87,13 +87,13 @@ const genStatusStyle = (
           [`${componentCls}-focused& ${componentCls}-selector`]: {
             borderColor: borderHoverColor,
             boxShadow: `0 0 0 ${token.controlOutlineWidth}px ${outlineColor}`,
-            borderInlineEndWidth: `${token.controlLineWidth}px !important`,
+            borderInlineEndWidth: `${token.lineWidth}px !important`,
```
Comment (afc163) [inline components/select/style/index.tsx:90]: 有空我看看能不能删掉，感觉没啥用。

Comment (afc163) [inline components/select/style/index.tsx:90]:
```suggestion
```

File: components/select/style/index.tsx [hunk 2]
```diff
@@ -64,7 +64,7 @@ const genStatusStyle = (
     borderHoverColor: string;
     outlineColor: string;
     controlOutlineWidth: number;
-    controlLineWidth: number;
+    lineWidth: number;
```
Comment (afc163) [inline components/select/style/index.tsx:67]:
```suggestion

```

File: components/select/style/index.tsx [hunk 3]
```diff
@@ -87,13 +86,12 @@ const genStatusStyle = (
...
             borderColor: borderHoverColor,
             boxShadow: `0 0 0 ${token.controlOutlineWidth}px ${outlineColor}`,
-            borderInlineEndWidth: `${token.controlLineWidth}px !important`,
             outline: 0,
           },
 
           [`&:hover ${componentCls}-selector`]: {
             borderColor: borderHoverColor,
-            borderInlineEndWidth: `${token.controlLineWidth}px !important`,
+            borderInlineEndWidth: `${token.lineWidth}px !important`,
```
Comment (afc163) [inline components/select/style/index.tsx:94]:
```suggestion
```

**Pull request merged**